### DISCUSSION TITLE
172

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#135] [General]
# 潮流周刊第172期 - 骑青山湖

封面图来源于周末去青山湖露营过程中骑行的路上所拍，骑了十几公里，这个地方有专门的骑行道，围着湖骑了一段距离，然后进入了延伸的河边，不热有风很舒服。

https://weekly.tw93.fun/posts/172-%E9%AA%91%E9%9D%92%E5%B1%B1%E6%B9%96

**Comment by zimiyu** [comment]
偷跑啦，，今天才4月14日...

**Reply by tw93** [reply]
哈哈哈哈 强迫症的周一

**Comment by Beeta** [comment]
Manner 有个「桂花拿铁」跟你的咖啡异曲同工

**Reply by tw93** [reply]
对对对，我挺喜欢桂花，包括大红袍加一点儿桂花，也超级好喝

**Reply by wukaipeng-dev** [reply]
@tw93 不错，那我也去试试大红袍桂花

**Comment by duofu** [comment]
最开始用SQLyog（功能简单），之后用Navicat（老崩溃，但胜在功能丰富、用数据同步工具），后来用Sequel-Ace（挺直观的、效率），最后用DataGrip，很香。。。

**Reply by tw93** [reply]
哈哈哈 你这叫做折腾之旅

**Comment by LookUpPls** [comment]
微信和支付宝支不支持消费后发送邮件或者调用接口

**Reply by tw93** [reply]
可以换一种思路，你的信用卡一般都支持邮件～
微信支付宝比较难搞
